keps/349-all-or-nothing/README.md

```diff
@@ -226,6 +232,14 @@ type WaitForPodsReady struct {
 	// RequeuingStrategy defines the strategy for requeuing a Workload.
 	// +optional
 	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+
+	// recoveryTimeout defines optional time duration, relative to
+	// pod's failure expressed by PodsReady=false condition, after a Workload is Admitted and running.
+	// After exceeding the timeout the corresponding job gets suspended again
+	// and moved to the ClusterQueue's inadmissibleWorkloads list. The timeout is
+	// enforced only if waitForPodsReady.enable=true. Defaults to 3 mins.
```

Comment:
```suggestion
	// recoveryTimeout defines an optional timeout, measured since the
	// last transition to the PodsReady=false condition after a Workload is Admitted and running.
	// Such a transition may happen when a Pod failed and the replacement Pod 
	// is awaited to be scheduled.
	// After exceeding the timeout the corresponding job gets suspended again
	// and requeued. The timeout is enforced only if waitForPodsReady.enable=true. 
	// Defaults to 3 mins.
```